docs/integrations/sources/shopify.md

```diff
@@ -99,6 +99,7 @@ This connector support both: `OAuth 2.0` and `API PASSWORD` (for private applica
 
 | Version | Date | Pull Request | Subject |
 | :--- | :--- | :--- | :--- |
+| 0.1.23 | 2021-11-09 | [7783](https://github.com/airbytehq/airbyte/pull/7783) | Fix schemas for missing `properties` and `items` |
```

Comment:
we should also add other stream names
```suggestion
| 0.1.23 | 2021-11-09 | [7783](https://github.com/airbytehq/airbyte/pull/7783) | Fix schemas for `discount_codes` stream |
```

Reply:
Actually this change touches almost all schemas, more or less. So, I'll put this:
```
| 0.1.23 | 2021-11-09 | [7783](https://github.com/airbytehq/airbyte/pull/7783) | Reviewed and corrected schemas for all streams |
```